draft-ietf-taps-interface.md

```diff
@@ -1613,10 +1613,10 @@ action is performed on the Listener object.
 Listener -> ConnectionReceived<Connection>
 ~~~
 
-The ConnectionReceived Event occurs when a Remote Endpoint has established a
+The ConnectionReceived Event occurs when a Remote Endpoint has established or cloned ((see {{groups}}) a
```

Comment:
Since the remote endpoint may not be using TAPS, I wonder if there's a better way to say this than that they "cloned" the connection — they are opening a connection or a new stream. Maybe we just say that?

Reply:
"…or cloned (e.g., by creating a new stream in a multi-stream transport; see {{groups}}), …" maybe? Otherwise, looks good.

Reply:
```suggestion
The ConnectionReceived Event occurs when a Remote Endpoint has established or cloned ((see {{groups}}) a
```